src/less/button/ds6/button-base.less

```diff
@@ -65,15 +67,11 @@ button.btn--primary,
 a.fake-btn--primary {
     background-color: @ds6-color-p002-blue;
     border: 1px solid @ds6-color-p002-blue;
```

Comment:
Nitpicking now, but now that we have a border type and thickness set in base class, we can just specify the border-color in the modifiers.